.github/workflows/test.yml

```
@@ -15,8 +15,8 @@ concurrency:
   cancel-in-progress: true
 
 env:
-  GO_VERSION: "1.20.6"
-  K8S_VERSION: v1.28.0
+  GO_VERSION: "1.21"
```

```suggestion
  GO_VERSION: "1.21.6"
```
? be consistent? pick either or , I see the previous file using one version and this one is doing another

---

k/k is on 1.21 so i will stick with this